### PR TITLE
Using GitHub forms for Issues

### PR DESCRIPTION
**Pull request by ruth3n**
GitHub forms can provide a more structured way to gather information when issues are created.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots to help explain your problem.
+    validations:
+      required: false
+  - type: input
```
Comment (zamitto) [inline .github/ISSUE_TEMPLATE/bug_report.yml:38]: could you update it so this type is a dropdown with the SO options?

**Review by ryeviz — COMMENTED** [review]
The form is well-structured, covering all necessary elements

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (ruth3n) [reply .github/ISSUE_TEMPLATE/bug_report.yml:38]: Will Windows and Linux be sufficient or do we need to specify the version/distribution?

Comment (piradata) [reply .github/ISSUE_TEMPLATE/bug_report.yml:38]: windows, linux and mac, and maube a optional field below so the user can say say the version (win7, 10, 11, or arch, on manjaro, or ubuntu, etc)

Comment (zamitto) [reply .github/ISSUE_TEMPLATE/bug_report.yml:38]: Yeah, when I commented I forgot about the version. I think the way you did is good for now

**Review by zamitto — APPROVED** [review]

**Pull request merged**